src/batch.html

```diff
@@ -0,0 +1,22 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Hoover</title>
+
+  <link rel="stylesheet" href="/bootstrap.css">
+  <link rel="stylesheet" href="/font-awesome.min.css">
+  <link rel="stylesheet" href="/style.css?t={{ timestamp }}">
```

Comment:
Perhaps a better cache strategy is needed for this and app.js? See [here](https://webpack.github.io/docs/long-term-caching.html).

Reply:
That looks like a really good idea for another PR.
We really have to lose those timestamps, so the page doesn't reload megabytes of css and js on every refresh / newtab / whatever

Reply:
Actually the timestamp is generated at build time. It serves to force cache invalidation whenever a build is done. So we don't really download assets on every page load.